bescort.lic

```diff
@@ -358,6 +364,16 @@ class Bescort
     end
   end
 
+  def asketis_mount(mode)
+    if mode.include?('up')
```

Comment:
Minor nitpicks, `mode` feels like an odd term as opposed to direction or something similar. Also include? feels like an odd choice here for matching a string when the fact that up/down is coming through parse args provides us some fairly strict guarantees.

Reply:
I thought `mode` was a little strange but it's what's used in all the other arg sets that take a second arg. I modeled my method partially off of [this one](https://github.com/rpherbig/dr-scripts/blob/7bff1eb59a7e84bb968d611678886c9974b04486/bescort.lic#L516).

You thinking something like
```ruby
if mode == /up/i
```

Reply:
`.match?` if using a regex is best practice at this point I believe, but you can just use `== "up"` I'm fairly certain arg parser forces exact matches and down cases them.